{"uuid": "2256da6b-32b5-43ef-b4af-d1a54b3d3264", "children": ["2b302c7c-bc74-46bb-a5be-930490883132"], "befores": [{"name": "xunit_setup_class_fixture_TestUserRegister", "status": "broken", "statusDetails": {"message": "xlrd.biffh.XLRDError: No sheet named <'TestUserRegister'>\n", "trace": "  File \"C:\\soft\\anaconda3\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 811, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\soft\\pycharm\\pythonProject\\test2\\request\\test_ydy\\test_register.py\", line 15, in setup_class\n    self.data_list = excel_to_list('test_user_data.xlsx', 'TestUserRegister')\n  File \"D:\\soft\\pycharm\\pythonProject\\test2\\request\\read_excel.py\", line 7, in excel_to_list\n    sh = wb.sheet_by_name(sheet)\n  File \"C:\\soft\\anaconda3\\lib\\site-packages\\xlrd\\book.py\", line 476, in sheet_by_name\n    raise XLRDError('No sheet named <%r>' % sheet_name)\n"}, "start": 1614927004401, "stop": 1614927004415}], "start": 1614927004401, "stop": 1614927004725}